plugins/woocommerce/changelog/dev-52568-settings-nav-link-to-use-non-reactified-page

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Do we need a changelog entry for this? Since it'll go into the `add/settings-nox` branch, and I guess we'll end up just having one changelog entry for the whole Reactified settings page.

Reply:
removing after slack discussion p1730810180012819-slack-C03KTTK2YMA